auto_nag/scripts/prod_comp_changed_with_priority.py

```diff
@@ -45,11 +46,13 @@ def history_handler(bug, data):
                                 prod = True
                             else:
                                 comp = True
+
             if (
                 priority_date is None
                 or prod_comp_date is None
                 or priority_date >= prod_comp_date
                 or priority_who == prod_comp_who
+                or prod_comp_who in self.skiplist
```

Comment:
Probably not common or at all depending on how often this runs, but prod_comp_who will get the latest changer of product/component potentially shadowing an earlier change by someone else.

Reply:
Fixed: https://github.com/mozilla/relman-auto-nag/pull/754